### ISSUE TITLE
Publish to central repo

### ISSUE THREAD

**Issue by mosabua** [casidiablo/android-facebook#1]
Have you considered publishing the artifact to central and potentially talked to somebody at facebook about it?

**Comment by casidiablo** [comment]
I have no experience doing that. Do you? If so... can you give me a hand?

El miércoles 29 de febrero de 2012, Manfred Moser <
reply@reply.github.com>
escribió:

**Comment by mosabua** [comment]
Yes I have and I can. I will update with details tomorrow..

**Comment by casidiablo** [comment]
Thanks dude. I really appreciate it.

**Comment by mosabua** [comment]
So I think as a first step we should try to convince facebook to upload themselves or grant us permission to do it with your package. If we get the permission we will be able to upload under the com.facebook groupId. I have done that before for google/android as part of the android4maven project. 

To facilitate that I will create an issue with the facebook github repo and see what happens. I will link them to this issue and your repo to show what sort of setup we would probably do. All would then have to do is agree on groupId, artifactId and version, build it with maven and use the repository plugin to create a signed bundle. I have done all that before and can do it again. Then I just upload it to the oss repo of sonatype and it gets synced to central... 

ok? If you are okay with this approach I will take the next steps..

**Comment by casidiablo** [comment]
That sounds perfect.

Also, there is another project I've been working on https://github.com/casidiablo/persistence which I would like to submit to central. I would like to do it by my self to learn the process... so it is just a matter of uploading it to Sonatype?

Thanks for your help and interest.

**Comment by mosabua** [comment]
If you are the copyright holder, it is open source  and want to publish under your groupId (you control that domain and can proof it e.g. via a email from it or whois record or whatever) .. you can just follow the deployment guide here

https://docs.sonatype.org/display/Repository/Sonatype+OSS+Maven+Repository+Usage+Guide

its easy.. and Juven can help if needed. I can too..

**Comment by mosabua** [comment]
Do you know if Facebook has some sort of developer issue tracker? I cant find any place to post a request to..

**Comment by casidiablo** [comment]
What about the issue tracker of the android sdk project here in GitHub? Wouldn't it be a good place to start?

**Comment by mosabua** [comment]
There is no issue tracker and even the pull request are not being looked at or pulled in at all.. seems nearly like a dead project..

**Comment by casidiablo** [comment]
You are right. The last commit was a month ago, so I think the project is not dead at all. However, the maintainer either does not know how to use GitHub or just does not like the idea of accepting pull requests.

There's this guy: Yariv Sadan yarivsblog@gmail.com... one of the latest committers with public email. Maybe we should ask him or something.

**Comment by mosabua** [comment]
I created this issue and will share this around to get some traction.

http://developers.facebook.com/bugs/345435392174808

It links back here too. We will see what comes out of that..

**Comment by mosabua** [comment]
Btw. I created a mavenized version of the whole sdk including the examples here 
https://github.com/mosabua/facebook-android-sdk

it is just upstream with some pom files added in the maven branch... all ready to go. I am using apklib as package for the sdk.

**Comment by casidiablo** [comment]
That's cool. At first I was avoiding apklib because the android-maven-plugin was still pretty buggy; that's why I replaced the drawables with a bitmap byte array. But now it worths the effort to use apklib.

**Comment by mosabua** [comment]
I sent an email to three fb.com developers from the git log now and will see what comes out of that..

**Comment by csgavino** [comment]
Any updates?

**Comment by jitpack** [comment]
Btw, android-facebook is available as a maven dependency on: https://jitpack.io/#casidiablo/android-facebook/222289fa98
